src/casp/services/api/services/cell_operations_service.py

```diff
@@ -17,6 +19,8 @@
 
 AVAILABLE_FIELDS_DICT = set(schemas.CellariumCellMetadata.__fields__.keys())
 
+GET_MATCHES_CHUNK_SIZE = 5
```

Comment:
Should we make this configurable?

Reply:
That's a good idea.  I could move it to the settings file and make it optionally configurable with an environment variable maybe?  I think that's the convention we're using currently.  Is that right @fedorgrab ?

Reply:
Okay cool.  I added it to the settings file